### PR TITLE
Feature: Update footer links

### LINKED ISSUE #25: Feature: update input and button components
### Type of feature

🍕 Feature

### Current behavior

input_newsletter

### Suggested solution

- Create a reusable input component inside `components/atoms`
- Change the button type (see screenshot)

:information_source: The button can be outside of the input for now

Final Result:

input_newsletter_xd

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### LINKED ISSUE #27: Feature: Update footer links
### Type of feature

🍕 Feature

### Current behavior

footer_links

### Suggested solution

Update links position and sizes to match the image below

footer_links_xd

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by dev-phantom**
## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] ⏩ Revert

## Description

Updated the footer
## Related Tickets & Documents

Fixes #27
Fixes #25
## Mobile & Desktop Screenshots/Recordings

image

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [x] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### 📸 Credits

This pull request template was created by [Open Sauced](https://github.com/open-sauced) 🍕

### CODE REVIEW

**Comment by dev-phantom** [comment]
sorry i  made a mistake in the branch name i though it was a fix

**Comment by dev-phantom** [comment]
done

File: apps/client/src/components/Footer.tsx [hunk 1]
```diff
@@ -20,24 +20,24 @@ const Footer = () => {
...
       </div>
       <div className="links flex gap-20">
-        <div className="link-container flex flex-col">
-          <h6>Explore</h6>
-          <Link href="/#">Feed</Link>
-          <Link href="/#">Trending Blogs</Link>
-          <Link href="/#">Tags</Link>
-          <Link href="/#">Search</Link>
+        <div className="link-container flex flex-col gap-3">
+          <h6 className="text-slate-400 font-bold text-[22px]">Explore</h6>
```
Comment (TAKANOME-DEV) [inline apps/client/src/components/Footer.tsx:24]:
```suggestion
          <h6 className="text-slate-400 font-bold text-xl">Explore</h6>
```

File: apps/client/src/components/Footer.tsx [hunk 2]
```diff
@@ -20,24 +20,24 @@ const Footer = () => {
...
+          <Link href="/#" className='text-lg'>Search</Link>
         </div>
-        <div className="link-container flex flex-col">
-          <h6>BlogHive</h6>
-          <Link href="/#">About</Link>
-          <Link href="/#">Contact</Link>
-          <Link href="/#">Terms</Link>
-          <Link href="/#">Privacy</Link>
+        <div className="link-container flex flex-col gap-3">
+          <h6 className="text-slate-400 font-bold text-[22px]">BlogHive</h6>
```
Comment (TAKANOME-DEV) [inline apps/client/src/components/Footer.tsx:31]:
```suggestion
          <h6 className="text-slate-400 font-bold text-xl">BlogHive</h6>
```

File: apps/client/src/components/Footer.tsx [hunk 3]
```diff
@@ -20,24 +20,24 @@ const Footer = () => {
...
+          <Link href="/#" className='text-lg'>Contact</Link>
+          <Link href="/#" className='text-lg'>Terms</Link>
+          <Link href="/#" className='text-lg'>Privacy</Link>
         </div>
-        <div className="link-container flex flex-col">
-          <h6>Support</h6>
-          <Link href="/#">Community</Link>
-          <Link href="/#">Discussion</Link>
+        <div className="link-container flex flex-col gap-3">
+          <h6 className="text-slate-400 font-bold text-[22px]">Support</h6>
```
Comment (TAKANOME-DEV) [inline apps/client/src/components/Footer.tsx:38]:
```suggestion
          <h6 className="text-slate-400 font-bold text-xl">Support</h6>
```

**Review by TAKANOME-DEV — CHANGES_REQUESTED** [review]

**Comment by dev-phantom** [comment]
fixed

**Comment by dev-phantom** [comment]
fixed

**Review by TAKANOME-DEV — COMMENTED** [review]
Ready to be merged :fire: 

Just one thing: since there is an issue for the subscribe section (text size and color), I would suggest to revert the changes made in `pages/index.tsx` file

**Comment by dev-phantom** [comment]
Ive  reverted  the changes

**Comment by TAKANOME-DEV** [comment]
Awesome :tada: Thanks for your contribution :rocket:

**Pull request merged**